src/Apps/MyCollection/myCollectionRoutes.tsx

```diff
@@ -86,6 +86,9 @@ export const myCollectionRoutes: AppRouteConfig[] = [
             }
           }
         `,
+        cacheConfig: {
+          force: true,
+        },
```

Comment:
Similar to the artwork detail page, we want to always have fresh data when editing an artwork.